<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionLoadFile"/>
    <addaction name="actionQuit"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionQuit_2"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadFile">
   <property name="text">
    <string>Load File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
